core/src/main/java/com/sequenceiq/cloudbreak/controller/BlueprintController.java

```diff
@@ -33,14 +33,14 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(BlueprintController.class);
```

Comment:
Should we rename this class as well?

Reply:
This issue is not about the public facing of Blueprints, we have separated issue to rename the public part